README.md

```diff
@@ -510,7 +521,7 @@ hl = {bg = 'oceanblue'},
 right_sep = 'slant_right'
 ```
 
-Not only that, you can add your own custom colors and separators through [the setup function](#the-setup-function) which allows you to just use the name of the color or separator to refer to it.
+Not only that, you can add your own custom colors and separators through [the setup function](#setup-function) which allows you to just use the name of the color or separator to refer to it.
```

Comment:
I think I should have changed this to become `the [setup function](#setup-function)`

Reply:
does changing it to that fix the issue?